convert/jenkinsjson/json/checkout_test.go

```diff
@@ -106,3 +106,135 @@ func mapToNode(raw map[string]interface{}, node *Node) error {
 
 	return nil
 }
+
+func TestExtractGitUrl(t *testing.T) {
+	parameterMap := map[string]any{
+		"scm": map[string]any{
+			"arguments": map[string]any{
+				"userRemoteConfigs": []any{
+					map[string]any{"url": "abc"},
+				},
+			},
+		},
+	}
+	parameterMapWithSubArguments := map[string]any{
```

Comment:
These variables are very similar, is there a way to remove the duplication here?